kolibri/plugins/coach/assets/src/views/common/QuestionsAccordion.vue

```diff
@@ -178,6 +181,11 @@
         type: Boolean,
         required: false,
       },
+      isCheckBoxVisible: {
```

Comment:
It does seem to me like we do _want_ this checkbox to be conditionalized, so I don't think this is wrong, but I'm a little bit confused as to why it's included with your changes in this PR. I don't think there are any "selectable" questions as part of this work specifically (for previewing exercises) so I'm not sure why this has been made right now.

Reply:
I think on the figma , the sidepanel showing the questions was to reuse the questionAccordion component , but the conditionalizing of the checkbox was not in place, making me think adding a prop to handle it would work.